[bug] pxc crashed with -o switch on pvc describe

**Describe the bug**

```
[vagrant@ip-70-0-52-122 ~]$ kubectl pxc pvc describe -n tenantone pvc-csi-auth -o wide
unknown shorthand flag: 'o' in -o
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x1c pc=0x908cb2]

goroutine 1 [running]:
github.com/portworx/pxc/pkg/util.(*SigIntManager).Stop(0x0, 0x0, 0x0)
        /tmp/pxc-20200707-26942-4uo281/src/github.com/portworx/pxc/pkg/util/sigint.go:75 +0x42
github.com/portworx/pxc/cmd.rootPersistentPostRunE(0x0, 0x2254098, 0x0, 0x0, 0xc00068ff58, 0x1)
        /tmp/pxc-20200707-26942-4uo281/src/github.com/portworx/pxc/cmd/root.go:157 +0x3a
github.com/portworx/pxc/cmd.Execute()
        /tmp/pxc-20200707-26942-4uo281/src/github.com/portworx/pxc/cmd/main.go:56 +0xe8
main.main()
        /tmp/pxc-20200707-26942-4uo281/main.go:24 +0x20
```

**Version**
v0.29.0

**Additional Information**

--- a/cmd/root.go
+++ b/cmd/root.go
@@ -154,7 +154,10 @@ func rootPersistentPreRunE(cmd *cobra.Command, args []string) error {
 func rootPersistentPostRunE(cmd *cobra.Command, args []string) error {
 	// Close the global tunnel if any
 	cleanup()
-	rootSignalHandler.Stop()
+
+	if rootSignalHandler != nil {
+		rootSignalHandler.Stop()
+	}
 
 	return nil
 }

--- a/handler/volume/patch.go
+++ b/handler/volume/patch.go
@@ -63,26 +63,18 @@ var _ = commander.RegisterCommandVar(func() {
 		Aliases: []string{"patch"},
 		Short:   "Update field(s) of a portworx volume",
 		Example: `
-  # Set the High Availability Level (HA Level) of the volume xyz to 3
-  pxc volume update xyz --halevel 3
+  #### Update Volume Spec ####
 
   # Update the size of the volume to 2GiB
   pxc volume update xyz --size 2
 
-  # Set the sticky flag of the volume xyz
-  pxc volume update xyz --sticky
-
   # Set the shared flag of the volume xyz
-  pxc volume update xyz --shared
+  pxc volume update xyz --shared=on
+
+  #### Update Volume Access Controls ####
 
   # Update collaborators and groups of the volume access list
   pxc volume update xyz --add-collaborators user1:r,user2:w,user3:a --add-groups group1:r,group2:w,group3:a
-
-  # Remove collaborators and groups from exisiting volume access list
-  pxc volume update xyz --remove-collaborators user1:r, --remove-groups group1:r
-
-  # Remove all the collaborators and groups from exisiting volume access list
-  pxc volume update xyz --remove-all-collaborators --remove-all-groups
 
   # Update collaborators and remove few groups from the volume access list
   pxc volume update xyz --add-collaborators user4:r,user5:w, --remove-groups group1:r

